src/bin/ccg.rs

```diff
@@ -358,6 +360,10 @@ fn write_to_file(mut output: String, name: String) {
     debug!("write to file: OK");
 }
 
+fn exact_vdisks_count() -> bool {
+    get_matches().is_present("exact_vdisks_count")
```

Comment:
I think it redundant to have one line fn with call from the only place

Reply:
It's here to avoid string constants in large functions, and in case of future reuse